src/main/java/jenkins/plugins/nodejs/tools/pathresolvers/LatestInstallerPathResolver.java

```diff
@@ -111,11 +117,19 @@ public String resolvePathFor(String version, Platform platform, CPU cpu) {
             break;
         case arm64:
         case armv6l:
-            if (nodeVersion.compareTo(new NodeJSVersion(12, 0, 0)) >= 0 || nodeVersion.compareTo(new NodeJSVersion(8, 6, 0)) == 0) {
+            if (nodeVersion.compareTo(new NodeJSVersion(12, 0, 0)) >= 0 || nodeVersion.compareTo(new NodeJSVersion(8, 6, 0)) == 0 //
+                || platform == Platform.AIX) {
                 throw new IllegalArgumentException(Messages.InstallerPathResolver_unsupportedArch(version, cpu.name(), platform.name()));
             }
         case armv7l:
-            if (nodeVersion.compareTo(new NodeJSVersion(4, 0, 0)) < 0) {
+            if (nodeVersion.compareTo(new NodeJSVersion(4, 0, 0)) < 0 //
+                || platform == Platform.AIX) {
```

Comment:
same above consideration.

Reply:
Hello, during the test done in the build step, I was facing some issues, so I wanted to be as secured as possible. This is same comment for 3 cases.